### ISSUE TITLE
Cocitations filter seems to be broken

### ISSUE THREAD

**Issue by IgorRodchenkov** [PathwayCommons/pcviz#27]
pcviz (both at pathwaycommons.baderlab.org/pcviz/, which is built using the latest sources and paxtools and the older http://www.pathwaycommons.org/pcviz/) displays edges having cocitations=0.

http://www.pathwaycommons.org/pcviz/#neighborhood/BRCA1

Is that ok (might I misunderstand how it should work)?

Attn: @armish @onursumer

**Comment by armish** [comment]
[That lower threshold is evaluated with a lower-than operator](https://github.com/PathwayCommons/pcviz/blob/master/src/main/java/org/pathwaycommons/pcviz/service/PathwayCommonsGraphService.java#L229). So essentially, a threshold set to 0 means that no filtering is in place. We used to have that set to 1; but, at least two users complained that the n-hoods of under-studied genes were showing up empty as often, we don't have publication support for those. 

So yes, you got it right; the filtering currently doesn't work but it is because we configured it as such.
